pkg/api/datasources.go

```diff
@@ -529,12 +475,6 @@ func (hs *HTTPServer) UpdateDataSourceByID(c *contextmodel.ReqContext) response.
 		return response.Error(http.StatusInternalServerError, "Failed to update datasource", err)
 	}
 
-	// check if LBAC rules have been modified
```

Comment:
removes the permissions checks

Reply:
If I'm not mistaken the `OnlyUpdateLBACRulesFromAPI` toggle being `false`, the rules won't be updated.

Reply:
@eleijonmarck to me `OnlyUpdateLBACRulesFromAPI` seems to be named counter-intuitively. It sounds like it would only allow updating LBAC rules, when it actually prevents updates. What do you think about renaming it to `AllowLBACRulesUpdates`?

Reply:
ah that sounds way better. il change it to `AllowLBACRulesUpdates` 👍